File: src/WinUEFI.py
import sys
import os

try:
    if os.name == 'nt':
        os.system('shutdown /fw /f /r /t 00')
    else:
        print('Not running on a Windows platform! Exiting...')
        sys.exit()
except Exception as e:
    print(f'An error occurred: {e}')
    sys.exit()

File: src/WinUEFI.js
const { execSync } = require('child_process');
const os = require('os');

if (os.platform() !== 'win32') {
  console.log('Not running on a Windows platform! Exiting...');
  process.exit(1);
} else {
  execSync('shutdown /fw /f /r /t 00');
}
